index.js

```diff
@@ -0,0 +1,50 @@
+var playItem = require('./src/playitem.js');
+
+var playlistMaker = function(player, plist) {
+  var currentIndex = 0;
+  var playlist = function playlist(list) {
+    this.playlist = playlistMaker(this, list);
+  };
+
+  playlist.list = function list() {
+    return plist.slice();
+  };
+
+  playlist.item = function item(index) {
```

Comment:
This is no `buffered`.
Does that mean that `player.playlist([1,2,3])` would then re-set the playlist?
```js
player.playlist([1,2,3]);
player.playlist(1); // select `2`
player.playlist([4,5,6,7,8,9]);
player.playlist(5); // select `9`
```